src/gennylib/src/Orchestrator.cpp

```diff
@@ -86,7 +86,7 @@ PhaseNumber Orchestrator::awaitPhaseStart(bool block, int addTokens) {
         for (auto&& cb : _prePhaseHooks) {
             cb(this);
         }
-        BOOST_LOG_TRIVIAL(debug) << "Beginning phase " << currentPhase;
+        BOOST_LOG_TRIVIAL(info) << "Beginning phase " << currentPhase;
```

Comment:
This was specifically requested in the ticket

Reply:
I think this is the only thing I'm slightly skeptical about. Right now phase transitions are pretty speedy, and I think @dalyd had mentioned before that there are workloads that run one phase, then transition to another, and try to measure the "hangover" from one phase to the next?

Not that printing a log line is the most expensive operation in the world or anything, but considering that it seemed to slow down unit tests enough to be noticeable in the last PR, there's likely some degree of effect here.

Main ask here is to run a faster workload like mixedworkloads and see if there's a visible difference. Sorry for the paranoia, I know it's annoying! Hopefully this sort of check will be easier as we improve patch-analysis workflows.

Reply:
Ran some patches with mixedworkloads and I think this should be fine (used the perf-viz app to do some comparisons)